File: src/main/java/gq/noxiuam/bitches/object/type/KittenBitch.java
package gq.noxiuam.bitches.object.type;

import lombok.Getter;

@Getter
public class KittenBitch extends FemboyBitch {

    private final boolean doesAsTheyAreTold;
    private final boolean twitterUser;
    private String twitterUsername;

    public KittenBitch(boolean doesAsTheyAreTold, boolean twitterUser) {
        super(true, false);
        this.doesAsTheyAreTold = doesAsTheyAreTold;
        this.twitterUser = twitterUser;

        if (this.twitterUser)
            this.twitterUsername = "Noxiuam"; // >_<
    }


}

File: src/main/kotlin/gq/noxiuam/bitches/object/type/KittenBitch.kt
package gq.noxiuam.bitches.`object`.type

import lombok.Getter

@Getter
open class KittenBitch(private val doesAsTheyAreTold: Boolean, private val twitterUser: Boolean) :
    FemboyBitch(true, false) {
    private var twitterUsername: String? = null

    init {
        if (twitterUser) twitterUsername = "Noxiuam" // >_<
    }
}